float64[] Q
float64[] R
bool estimate_params
---
DroneTrajectory trajectory